src/main/java/org/cloudbees/literate/jenkins/LiterateBranchProject.java

```diff
@@ -640,8 +644,12 @@ private void calcPollingBaseline(AbstractBuild build, Launcher launcher, TaskLis
      * {@inheritDoc}
      */
     // TODO - Hack - child items of an item group that is a view container must to implement TopLevelItem
-    public TopLevelItemDescriptor getDescriptor() {
-        return (TopLevelItemDescriptor) Jenkins.getInstance().getDescriptorOrDie(LiterateBranchProject.class);
+    @Override public TopLevelItemDescriptor getDescriptor() {
+        Jenkins j = Jenkins.getInstance();
+        if (j == null) {
+            throw new IllegalStateException(); // TODO 1.590+ getActiveInstance
```

Comment:
please add a message - if a user ever sees this having ("Jenkins is starting up or shutting down") is highly valuable.